### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by offhub**

### CODE REVIEW

File: SandboxiePlus/SandMan/Troubleshooting/UI/desktop.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ let form1 = [
   {type: 'box', id: 'box', name: tr('Select Sandbox')}
 ];
 
-let data1 = wizard.showForm(form1, tr('Sellect which  browser you want to create a shortcut for and in what box'));
+let data1 = wizard.showForm(form1, tr('Select which  browser you want to create a shortcut for and in what box'));
```
Comment (isaak654) [inline SandboxiePlus/SandMan/Troubleshooting/UI/desktop.js:48]:
```suggestion
let data1 = wizard.showForm(form1, tr('Select which browser you want to create a shortcut for and in what box'));
```

File: SandboxiePlus/SandMan/Wizards/SetupWizard.cpp [hunk 2]
```diff
@@ -583,23 +583,23 @@ CSBUpdate::CSBUpdate(QWidget *parent)
...
-    //m_pTemplates->setToolTip(tr("Check for latest compatybility tempaltes."));
+    //m_pTemplates = new QCheckBox(tr("Keep Compatibility Templates up to date"));
+    //m_pTemplates->setToolTip(tr("Check for latest compatibility tempaltes."));
     //layout->addWidget(m_pTemplates, row++, 1, 1, rows-1);
     //registerField("updateCompat", m_pTemplates);
 
-    m_pHotfixes = new QCheckBox(tr("Keep Compatybility Templates up to date and apply hotfixes"));
-    m_pHotfixes->setToolTip(tr("Check for latest compatybility tempaltes and hotfixes."));
+    m_pHotfixes = new QCheckBox(tr("Keep Compatibility Templates up to date and apply hotfixes"));
+    m_pHotfixes->setToolTip(tr("Check for latest compatibility tempaltes and hotfixes."));
```
Comment (isaak654) [inline SandboxiePlus/SandMan/Wizards/SetupWizard.cpp:592]:
```suggestion
    m_pHotfixes->setToolTip(tr("Check for latest compatibility templates and hotfixes."));
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: SandboxiePlus/SandMan/Wizards/BoxAssistant.cpp [hunk 3]
```diff
@@ -986,7 +986,7 @@ CSubmitPage::CSubmitPage(QWidget *parent)
     pLayout->addWidget(m_pReport, row++, 0, 1, 3);
 
     m_pAttachIni = new QCheckBox(tr("Attach Sandboxie.ini"));
-    m_pAttachIni->setToolTip(tr("Sandboxing compatybility is relyent on the configuration hence attaching the sandboxie.ini helps a lot with finding the issue."));
+    m_pAttachIni->setToolTip(tr("Sandboxing compatibility is reliant on the configuration hence attaching the sandboxie.ini helps a lot with finding the issue."));
```
Comment (isaak654) [inline SandboxiePlus/SandMan/Wizards/BoxAssistant.cpp:989]:
```suggestion
    m_pAttachIni->setToolTip(tr("Sandboxing compatibility is reliant on the configuration hence attaching the Sandboxie.ini file helps a lot with finding the issue."));
```

File: SandboxiePlus/SandMan/Troubleshooting/Sandboxing/installer.js [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 let form = [
     { id: 'path', name: tr('Enter Installer Path'), type: 'file' },
-    { id: 'box', name: tr('Sellect a sandbox to instll into'), type: 'box' },
+    { id: 'box', name: tr('Select a sandbox to instll into'), type: 'box' },
```
Comment (isaak654) [inline SandboxiePlus/SandMan/Troubleshooting/Sandboxing/installer.js:11]:
```suggestion
    { id: 'box', name: tr('Select a sandbox to install into'), type: 'box' },
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — CHANGES_REQUESTED** [review]
More fixes to be merged where applicable (from the private repo):
https://github.com/DavidXanatos/Sandboxie-Next/commit/e871ae7d92f04637e64733db6ffad981f12f4ded
https://github.com/DavidXanatos/Sandboxie-Next/commit/7389574785e46a570fb578c16a4ae07890bbffc2
https://github.com/DavidXanatos/Sandboxie-Next/commit/78267d98262513e4f8a6ad594fc0f4823f31ff9d
https://github.com/DavidXanatos/Sandboxie-Next/commit/632a6345ef7fc73a79a4c1fae50b27c314cc38db
https://github.com/DavidXanatos/Sandboxie-Next/commit/afeacb8c3b8ebad86e6b6bd802e4608c91d8ce6f
https://github.com/DavidXanatos/Sandboxie-Next/commit/35afd91590be7ad0cdd8730e32510626fd57f22c

**Comment by isaak654** [comment]
@DavidXanatos 
The following string updates to the `SettingsWindow.ui` file were suddenly reverted without notice: https://github.com/sandboxie-plus/Sandboxie/commit/fd73b2fa1fee6cdc58650b238eaefb586487a96e https://github.com/sandboxie-plus/Sandboxie/commit/1eccc1a9cf2b1f96adfd8d35b3ba0878f13f0e6f https://github.com/sandboxie-plus/Sandboxie/commit/c2a036ef4aef1bdd76dd4d0a057e8fdfebea5dbd

I'm aware of the issues you have with Qt Designer, so please make sure to apply manually the `SettingsWindow.ui` updates above or I'll have to abort this PR accordingly.

**Comment by DavidXanatos** [comment]
changes re applied

**Pull request merged**